Navaids/FIXES_Non-UK.txt

```diff
@@ -1,152 +1,42 @@
-BOLTA N54.49.27.00 E4.11.09.99
-BORVE N57.48.00.00 W2.28.00.00
-BUSBI N53.32.24.00 W2.6.47.99
-CAMBL N55.53.30.00 W4.1.00.00
-CAMBY N53.53.34.99 W3.29.39.00
-CHEDH N51.46.59.99 W1.55.00.00
-DAVWP N51.58.01.80 W1.43.05.40
-DEXET N54.46.51.99 W11.4.24.99
-DIXIS N45.0.00.00 W10.0.00.00
-DOMIG N53.32.13.48 W1.52.59.63
-DOPOL N57.21.04.60 W4.14.18.36
-EAGLE N51.24.27.00 E0.5.09.99
-EASTG N53.24.28.09 W1.51.11.79
-ELPIN N56.0.00.00 W12.0.00.00
-EMJEE N54.22.42.99 E1.52.42.99
-EPEKO N56.32.48.00 E3.12.42.00
-ETLEY N53.9.24.00 W2.27.42.00
-GATSU N51.10.52.32 E0.3.05.46
-GELKI N53.59.48.99 W5.54.15.00
-GERGA N51.35.15.99 W0.30.48.99
-GODAL N51.16.31.99 W0.44.33.99
-GOMLA N55.34.46.99 E4.45.31.99
-GONAN N045.00.00.000 W014.00.00.000
-GOSFR N55.59.31.39 W3.1.01.87
-INGEN N51.2.11.69 W0.13.14.40
-INRUT N56.43.39.00 E2.55.19.99
-KENUK N50.0.00.00 W12.0.00.00
-KESON N52.32.25.99 E1.44.13.99
-KILBA N51.32.39.99 W0.11.49.99
-KIRKY N55.57.24.00 W4.5.17.99
-LAPKU N51.59.42.10 W1.54.29.45
-LASNO N048.35.54.000 W009.00.00.000
-LETSI N50.51.38.61 W3.7.05.05
-LICWP N52.34.01.20 W2.25.04.80
-LUMEK N50.25.49.99 W3.13.33.99
-LUTON N51.51.07.99 W0.20.30.00
-MAKAM N54.50.17.99 W1.40.21.00
-MARIL N51.35.37.99 W0.48.31.99
-MARLI N55.18.06.00 E4.22.12.99
-MCHNB N53.21.10.99 W2.16.13.99
-MELOR N53.26.12.00 W1.56.48.00
-MILDE N53.35.30.99 W0.15.15.99
-MOSEN N55.49.02.70 W4.0.29.04
-MULLA N54.11.12.99 W5.44.51.00
-NAMON N56.18.06.99 E4.36.10.99
-NEMAX N52.50.07.80 W1.4.57.49
-NILAV N045.00.00.000 W013.25.00.000
-NUBLU N55.1.01.99 W1.46.37.99
-NUSVI N57.22.28.98 W4.24.34.33
-ODLUM N55.0.00.00 W12.0.00.00
-OKTIR N55.43.16.99 E4.48.06.99
-OMIRI N56.28.57.99 E4.54.39.99
-OMOKO N048.50.20.000 W012.00.00.000
-OSBAR N56.4.48.99 E4.13.48.99
-OTRAL N56.20.39.00 E4.16.18.99
-PEKOX N51.47.27.99 W1.12.15.99
-PEMAD N55.58.59.99 E4.34.52.99
-PENAV N51.27.55.71 W0.10.01.89
-PITAX N45.0.00.00 W12.0.00.00
-RINGU N57.35.37.92 W3.41.47.04
-RODOG N51.4.46.99 E0.5.39.30
-RUSEL N59.0.00.00 W2.34.00.00
-SELSI N50.54.29.99 W0.48.42.00
-SOPNO N51.5.30.09 E0.10.58.39
-SOTED N51.55.04.99 W0.55.22.99
-STOCK N53.32.04.99 W1.49.15.99
-SUNEX N55.31.54.00 E4.54.24.00
-SUSAX N50.57.53.82 W0.28.40.82
-TAMEL N048.43.43.000 W010.29.50.000
-TARGT N55.44.35.88 W3.22.11.11
-TBRAX N55.46.13.32 W3.36.50.10
-TEMPL N55.52.43.07 W3.4.02.89
-TEVMO N54.41.34.86 W1.4.58.05
-TIBKO N55.59.42.99 E4.45.21.99
-TILOK N55.56.36.71 W4.5.00.39
-TINVA N53.33.00.06 W2.4.46.75
-TOAKS N51.50.12.00 E0.35.55.99
-TORAN N55.58.41.99 W4.15.00.00
-TRAFA N53.21.36.00 W2.38.17.99
-TUPLA N54.41.54.43 W1.17.08.61
-TURSA N50.51.11.29 E0.26.54.19
-TYRAA N55.42.59.49 E4.45.01.17
-ULANU N56.49.07.99 E2.30.20.99
-UVNOP N51.56.58.92 W2.30.05.91
-VAGAX N55.59.22.99 E4.52.41.99
-VALAL N56.16.40.99 E3.23.33.99
-VALIX N56.27.33.99 E3.33.00.00
-VESUV N55.43.00.00 E4.45.00.99
-WINSR N51.27.49.99 W0.46.24.99
-XETLA N59.59.30.99 E0.0.54.00
-XXXXX N51.5.02.16 W0.38.54.27
-
 ;Unnamed ATS route fixes
-AbmVANIN N053.58.42.000 W004.02.50.000
```

Comment:
AbmVANIN and other deleted in 'Unnamed ATS route fixes' have been replaced by 5LNCs